packages/components/atoms/f-button/src/components/Button.vue

```diff
@@ -135,6 +166,10 @@ export default {
             if (!VALID_BUTTON_SIZES.includes(this.buttonSize)) {
                 throw new TypeError(`buttonSize is set to "${this.buttonSize}", but it can only be one of the following buttonSizes: "${VALID_BUTTON_SIZES.join('", "')}"`);
             }
+
+            if (!VALID_BUTTON_ICON_POSITION.includes(this.hasIcon)) {
+                throw new TypeError(`hasIcon is set to "${this.hasIcon}", but it can only be one of the following buttonSizes: "${VALID_BUTTON_ICON_POSITION.join('", "')}"`);
```

Comment:
Super small thing – but should probably say:

"but it can only be one of the following **values**"

rather than `buttonSizes`